RELEASE_NOTES.md

```diff
@@ -2,19 +2,20 @@
 
 ## 12.0.6 (TBD)
 
-The following items are new since Chef Server 12.0.4 and/or are changes from previous versions.
+The following items are new since Chef Server 12.0.5 and/or are changes from previous versions.
 For specific breakdown of updated components, refer to CHANGELOG.md
 
 ### opscode-omnibus
 * Use a cert instead of a public key for the superuser.
 * No longer generate /etc/opscode/pivotal.cert as it is no longer used. Now the public key for the superuser lives in the database and no longer exists on disk.
-* oc\_erchef
-  * Key API Updates - GET, PUT and DELETE  of named keys now supported
-  * Policyfile validation support. (See API Changes, below.)
 
-### Closed Issues
+### oc\_erchef
+* Key API Updates - GET, PUT and DELETE  of named keys now supported
+* Policyfile validation support. (See API Changes, below.)
 
-*
+### Closed Issues
+* [chef-server-115][https://github.com/chef/chef-server/issues/115]
```

Comment:
Yeah, was making this consistent with previous releases, but I like links myself. Will fix up